src/utilities/color-utils.ts

```diff
@@ -0,0 +1,17 @@
+import colorString from "color-string";
+
+export const luminanceRGB = (r: number, g: number, b: number, scale = 1) => {
+  // https://en.wikipedia.org/wiki/Relative_luminance
+  return (r * 0.2126 + g * 0.7152 + b * 0.0722) / scale;
+};
+
+export const luminanceColorString= (color: string) => {
+  const rgb = colorString.get.rgb(color);
+  return rgb && luminanceRGB(rgb[0], rgb[1], rgb[2], 255);
+};
+
+export const isLightColorRequiringContrastOffset = (color?: string) => {
+  const kLightLuminanceThreshold = 0.85;
+  const luminance = color && luminanceColorString(color);
+  return (luminance != null) && (luminance >= kLightLuminanceThreshold);
+};
```

Comment:
Somewhat related (and part of a different discussion), but I had to build some similar color util functions in the Activity Player.  Maybe in the future these sorts of utils could be unified?  Just thinking out loud.
https://github.com/concord-consortium/activity-player/blob/master/src/utilities/theme-utils.ts

Reply:
I think I looked at those but felt that they didn't seem quite on point for my use case. You might consider using the `color-string` library for AP so you don't have to do the parsing yourself.

Reply:
Agree.  Just wondering if at some point we have a bunch of projects that have util functions like this and if they belong in some general purpose location (probably the translation work has me thinking in this direction).  Anyway, nothing here in CLUE needs to change.